gramex/data.py

```diff
@@ -13,6 +13,8 @@
 from tornado.escape import json_encode
 from gramex.config import merge, app_log
 from orderedattrdict import AttrDict
+from bson.objectid import ObjectId
```

Comment:
Don't import bson at the top level. Gramex does not require MongoDB. If it's there, we use it. So import it inside the MongoDB filter functions where required.